### PR TITLE
vt: add page

### PR DESCRIPTION
**Pull request by kbdharun**
Signed-off-by: K.B.Dharun Krishna <kbdharunkrishna@gmail.com>

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** [v.0.10.4](https://github.com/VirusTotal/vt-cli/releases/tag/0.10.4)

### CODE REVIEW

File: pages/common/vt.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# vt
+
+> VirusTotal Command Line Interface.
+> API key from a VirusTotal account is required for this command.
+> More information: <https://github.com/VirusTotal/vt-cli>.
+
+- Scan a file for viruses:
+
+`vt scan file {{file}}`
```
Comment (navarroaxel) [inline pages/common/vt.md:9]:
```suggestion
`vt scan file {{path/to/file}}`
```

File: pages/common/vt.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# vt
+
+> VirusTotal Command Line Interface.
+> API key from a VirusTotal account is required for this command.
+> More information: <https://github.com/VirusTotal/vt-cli>.
+
+- Scan a file for viruses:
```
Comment (navarroaxel) [inline pages/common/vt.md:7]:
```suggestion
- Scan a specific file for viruses:
```

File: pages/common/vt.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+`vt scan file {{file}}`
+
+- Scan a URL for viruses:
+
+`vt scan url {{url}}`
+
+- Display analysis information:
```
Comment (navarroaxel) [inline pages/common/vt.md:15]:
```suggestion
- Display information from a specific analysis:
```

File: pages/common/vt.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+`vt analysis {{f-id|u-id}}`
```
Comment (navarroaxel) [inline pages/common/vt.md:17]:
```suggestion
`vt analysis {{file_id|analysis_id}}`
```

File: pages/common/vt.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+- Download files in encrypted `.zip` format (requires premium account):
+
+`vt download {{id}} --output {{path/to/directory}} --zip --zip-password {{string}} `
```
Comment (navarroaxel) [inline pages/common/vt.md:21]:
```suggestion
`vt download {{file_id}} --output {{path/to/directory}} --zip --zip-password {{password}}`
```

File: pages/common/vt.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+- Initialize or re-initialize `vt` to enter API key:
+
+`vt init`
+
+- Display information about a domain:
+
+`vt domain {{URL}}`
```
Comment (navarroaxel) [inline pages/common/vt.md:29]:
```suggestion
`vt domain {{url}}`
```

File: pages/common/vt.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+- Display analysis information:
+
+`vt analysis {{f-id|u-id}}`
+
+- Download files in encrypted `.zip` format (requires premium account):
+
+`vt download {{id}} --output {{path/to/directory}} --zip --zip-password {{string}} `
+
+- Initialize or re-initialize `vt` to enter API key:
```
Comment (navarroaxel) [inline pages/common/vt.md:23]:
```suggestion
- Initialize or re-initialize `vt` to enter API key interactively:
```

File: pages/common/vt.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+`vt init`
+
+- Display information about a domain:
+
+`vt domain {{URL}}`
+
+- Display information for a specific URL:
+
+`vt url {{URL}}`
```
Comment (navarroaxel) [inline pages/common/vt.md:33]:
```suggestion
`vt url {{url}}`
```

File: pages/common/vt.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+- Display information for a specific ipaddress:
```
Comment (navarroaxel) [inline pages/common/vt.md:35]:
```suggestion
- Display information for a specific IP address:
```

**Review by einverne — APPROVED** [review]
Looks good after @navarroaxel 's suggestion.

File: pages/common/vt.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+`vt domain {{url}}`
+
+- Display information for a specific URL:
+
+`vt url {{url}}`
+
+- Display information for a specific IP address:
+
+`vt domain {{ip}}`
```
Comment (navarroaxel) [inline pages/common/vt.md:37]:
```suggestion
`vt domain {{ip_address}}`
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**